README.md

```diff
@@ -7,38 +7,76 @@
+## First task: study `/lottery/lucky`
 
-## Debug static routes
+This app currently supports one URL: `/lottery/lucky`
 
-I've added a list of nav links to `/zodiacs/leo`, `/zodiacs/cancer`, etc.
+Follow the R→C→A→V through and study how it works, from `config/routes.rb` through the `app/controllers/` folder, and finally to a template within `app/views/`. Use the [RPS RCAV slides](https://slides.com/raghubetina/06-routing-rcav?token=43w7FD8Q) and [Routing — RCAV chapter](https://chapters.firstdraft.com/chapters/779) as a reference.
 
-Currently, none of them work. In `config/routes.rb`, you'll see that I've added 12 routes but commented them all out. Each RCAV is broken in some way.
+Notice that this action is using a custom controller class, rather than the out-of-the-box `ApplicationController`. This helps us keep things organized in larger apps; we can create as many as we like, and have them _inherit_ from `ApplicationController`.
 
-Uncomment each one *ONE AT A TIME* and make it work.
+```ruby
+# app/controllers/numbers_controller.rb
 
-Let me say that again:
+class NumbersController < ApplicationController
+  # ...
+```  
+
+`app/views/lottery_stuff/woohoo.html.erb` demonstrates how to do a `.each` loop within a `.html.erb` embedded Ruby template. Ask a question about _anything_ that you don't understand about this R→C→A→V.
 
-#### Uncomment each route ONE AT A TIME and debug it
+## Part 1: Build `/lottery/unlucky`
 
-**ONE**
+Implement the URL `/lottery/lucky`.
 
-**AT**
+See [the target](https://fortune-teller.matchthetarget.com) for behavior.
 
-**A**
+## Part 2: Debug zodiacs
 
-**TIME**
+In the nav, there are links to `/zodiacs/leo`, `/zodiacs/cancer`, etc; one for each of the [twelve Western zodiac signs](https://en.wikipedia.org/wiki/Astrological_sign#Western_zodiac_signs).
 
-If you uncomment them all at once, you'll have lots of problems because they all have bugs in them.
+Currently, none of them work. In `config/routes.rb`, you'll see that I've added 12 routes but commented them all out. Each RCAV is broken in some way(s).
 
-Refer frequently to the [Routing Chapter](https://chapters.firstdraft.com/chapters/772).
+Uncomment each one *ONE AT A TIME* and make it work like the target.
+
+Let me say that again:
 
-I've planted at least one bug into each RCAV.
+**Uncomment each route ONE AT A TIME and debug it ONE AT A TIME.** Don't uncomment multiple at once, or you'll have multiple syntax errors at once and the error messages can't help you.
 
 **YOUR JOB:** Debug all 12 RCAVs.
+
+Debugging checklist:
+
+ 1. READ the error message. Extract as much useful information from it as possible.
+ 2. If there's no error message, find another way to give yourself feedback; **make the invisible visible**.
+    - Use the server log.
+    - Print things; in this new world, that means use embedded Ruby tags (`<%=  %>`) in the view templates. We can't use the `p` method anymore since we aren't writing command line programs anymore.
```

Comment:
I think we should reword this line because technically they still can see the `p` in the server log.